src/control/cmd/dmg/pool.go

```diff
@@ -191,6 +192,73 @@ func (r *PoolExcludeCmd) Execute(args []string) error {
 	return err
 }
 
+// PoolExtendCmd is the struct representing the command to Extend a DAOS pool.
+type PoolExtendCmd struct {
+	logCmd
+	ctlInvokerCmd
+	UUID  string `long:"pool" required:"1" description:"UUID of the DAOS pool to extend"`
+	Ranks string `long:"ranks" required:"1" description:"Comma-separated list of ranks to add to the pool"`
+	// Everything after this needs to be removed when pool info can be fetched
+	GroupName string `short:"g" long:"group" description:"DAOS pool to be owned by given group, format name@domain"`
+	UserName  string `short:"u" long:"user" description:"DAOS pool to be owned by given user, format name@domain"`
+	ACLFile   string `short:"a" long:"acl-file" description:"Access Control List file path for DAOS pool"`
+	ScmSize   string `short:"s" long:"scm-size" required:"1" description:"Size of SCM component of DAOS pool"`
+	NVMeSize  string `short:"n" long:"nvme-size" description:"Size of NVMe component of DAOS pool"`
+	Sys       string `short:"S" long:"sys" default:"daos_server" description:"DAOS system that pool is to be a part of"`
+	// END TEMPORARY SECTION
+}
+
+// Execute is run when PoolExtendCmd subcommand is activated
+func (e *PoolExtendCmd) Execute(args []string) error {
+	msg := "succeeded"
+
+	var ranklist []uint32
```

Comment:
changes to rank parsing (to support range patterns) in https://github.com/daos-stack/daos/pull/2730 but looks like your PR will land first so I will most likely fix in my PR.